tests/testthat/test-renderer2-PredictedPeaks.R

```diff
@@ -8,17 +8,35 @@ writeBin(content(request), PredictedPeaks.RData)
 ## If we don't load this data set into the global environment, then we
 ## get Error in eval(expr, envir, enclos) (from helper-functions.R#5)
 ## : object 'PredictedPeaks' not found
-load(PredictedPeaks.RData, .GlobalEnv) 
-Sys.sleep(0.5)
-
+load(PredictedPeaks.RData, .GlobalEnv)
+# Taking minimal subsets
+selected_chroms <- c("chr16", "chrM", "chrY")
+selected_types <- c("neutro", "mono")
+PredictedPeaks$chromCounts <- subset(PredictedPeaks$chromCounts, 
+                                   chrom %in% selected_chroms & 
+                                   type %in% selected_types & 
+                                   samples.up %in% c(1, 14, 38))
+PredictedPeaks$countsByChrom <- subset(PredictedPeaks$countsByChrom,
+                                     chrom %in% selected_chroms)
+PredictedPeaks$chrom.ranges <- subset(PredictedPeaks$chrom.ranges,
+                                    chrom %in% selected_chroms)
+PredictedPeaks$scatter.text <- subset(PredictedPeaks$scatter.text,
+                                    chrom %in% selected_chroms)
+PredictedPeaks$bg.rect <- subset(PredictedPeaks$bg.rect,
+                               chrom %in% selected_chroms & 
+                               nonInputType %in% selected_types &
+                               up %in% c(1, 14, 38))[1:50,]

```

Comment:
can you please make a copy of the data set (instead of modifying the original), and change the indentation to below?
```r
SomePeaks <- PredictedPeaks
SomePeaks$bg.rect <- subset(
  SomePeaks$bg.rect,
  chrom %in% selected_chroms
)[1:50,]
```
etc?